client/extensions/woocommerce/state/action-types.js

```diff
@@ -82,6 +82,9 @@ export const WOOCOMMERCE_SETTINGS_GENERAL_REQUEST = 'WOOCOMMERCE_SETTINGS_GENERA
 export const WOOCOMMERCE_SETTINGS_GENERAL_RECEIVE = 'WOOCOMMERCE_SETTINGS_GENERAL_RECEIVE';
 export const WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST = 'WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST';
 export const WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST_SUCCESS = 'WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST_SUCCESS';
+export const WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_ERROR = 'WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_ERROR';
+export const WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_REQUEST = 'WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_REQUEST';
+export const WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_SUCCESS = 'WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_SUCCESS';
```

Comment:
So far the pattern I have been going with is a `*_REQUEST` action for the initial request action, and a `*_RECEIVE` action that accepts either `error` or the data. 

Only reviews and general settings are doing it so far, but it would be good to be consistent. There was a super brief Slack 'discussion' on it (p1501170048976308-slack-woocommerce-on-wpcom) a couple months ago, but I believe that's what we settled with.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/17922/commits/427f3944f4b8a58d59ac7785e0695175bad9aff0